test_dagmc_stats.py

```diff
@@ -7,17 +7,74 @@
 
 # import the new module that defines each of the functions
 import dagmc_stats
-
+from collections import OrderedDict
+import statistics
 import nose
-
+import random as rd
 test_input = "3vols.h5m"
 
 my_core = core.Core()
 my_core.load_file(test_input)
-
+test_set = {}
+for x in range(rd.randint(1,10)):
+    test_set[rd.randint(10*x,10*x+10)] = rd.randint(1,10000)
 def test_get_tags():
 
     dagmc_tags = dagmc_stats.get_dagmc_tags(my_core)
     assert(len(dagmc_tags) == 3)
 
+def test_get_median():
+
+    # Sort the dictionary
+    values_sorted = OrderedDict(sorted(test_set.items(), key=lambda t: t[0]))
+    index = sum(values_sorted.values())/2
```

Comment:
Should not median calculation based on the number of occurrences in the set instead of their values ?

In a sorted set of values:
- if odd number of occurrences: median is the middle one:
        `(len(values_sorted)-1)/2`
- if even number of occurrences: median is the mean of the 2 middle values:
        `(len(values_sorted)/2-1` & `(len(values_sorted)/2+1`)